src/PasswordField.js

```diff
@@ -118,20 +117,18 @@ class PasswordField extends React.Component {
     const actualErrorText = errorText || hintText
 
     return (
-      <div style={{ ...styles.root, ...style }}>
-        <div style={styles.inputContainer}>
-          <TextField
-            {...other}
-            errorStyle={{...styles.error, ...errorStyle}}
-            errorText={errorText}
-            hintText={null}
-            style={{ ...styles.input, ...textFieldStyle }}
-            type={visible ? 'text' : 'password'}
-            onFocus={(event) => this.handleInputFocus(event)}
-            onBlur={(event) => this.handleInputBlur(event)}
-          />
-          {hintText && !errorText ? <div style={prepareStyles(styles.hint)}>{actualErrorText}</div> : null}
-        </div>
+      <div style={styles.root}>
+        <TextField
+          {...other}
+          errorStyle={{...styles.error, ...errorStyle}}
+          errorText={errorText}
+          hintText={null}
+          style={{ ...styles.input, ...textFieldStyle, ...style }}
```

Comment:
`style` is supposed to extend the style of the root component (the `div`), as it did before. Could you please restore the previous behavior?